### DISCUSSION TITLE
Exception: data did not match any variant of untagged enum ModelWrapper at line 1251003 column 3

### DISCUSSION THREAD

**Discussion by yuchenxie4645** [axolotl-ai-cloud/axolotl#2011] [Q&A]
After fine tuning an llama 3.2 3b with my custom alpaca dataset using axolotl using this command
accelerate launch -m axolotl.cli.train examples/openllama-3b/lora.yml
Then merging the lora weights with the main weights using this command:
python3 -m axolotl.cli.merge_lora your_config.yml --lora_model_dir="./completed-model"

I get the error:
Exception: data did not match any variant of untagged enum ModelWrapper at line 1251003 column 3

Obviously I had to transform the lora.yml so that it suites my need, but I used that as a guideline and workflow as to how I encountered this error.

**Comment by NanoCode012** [comment]
Hey @yuchenxie4645 , would you be able to provide the stack trace?
